src/js/src/components/SearchBox.vue

```diff
@@ -209,31 +211,25 @@ export default {
     selectSearchMethod(selected) {
       console.log(selected)
       if(selected === 'imgSearch') {
-        this.futureImgSearch = !this.futureImgSearch
-        this.futureImgSearch === true ? this.futureUrlSearch = false : null
-        //this.solrSettings.urlSearch ? this.updateSolrSettingUrlSearch(!this.solrSettings.urlSearch) : null
-        //this.updateSolrSettingImgSearch(!this.solrSettings.imgSearch)
+        this.updateFutureSolrSettingImgSearch(!this.futureSolrSettings.imgSearch)
```

Comment:
If the argument (this.futureSolrSettings.imgSearch) always should evaluate to _false_ then it is better to just give _false_ as argument. If the argument should be the Boolean opposite of _this.futureSolrSettings.imgSearch_ then this is fine as is.

Reply:
This is intended behavior. The selected checkbox should simply flip to the opposite of that it is now. So i'll leave this as it is.

Reply:
👍